tests/test_preprocess.py

```diff
@@ -124,3 +132,44 @@ def test_extend_lines(self):
         ext5 = mm.extend_lines(gdf, 2)
         assert ext5.length.sum() > gdf.length.sum()
         assert ext5.length.sum() == pytest.approx(6.2, rel=1e-3)
+
+    def test_roundabout_simplification_1(self):
```

Comment:
Can you rename these and instead of `_1` `_2` `_3` use something like `_default`, `_high_threshold`, `_low_threshold`, so we know what aspect is the test checking from the name?